core/src/main/java/org/apache/carbondata/core/util/CarbonProperties.java

```diff
@@ -855,7 +855,11 @@ private void loadProperties() {
    * Return the store path
    */
   public static String getStorePath() {
-    return getInstance().getProperty(CarbonCommonConstants.STORE_LOCATION);
+    String storePath = getInstance().getProperty(CarbonCommonConstants.STORE_LOCATION);
+    if (storePath == null) {
```

Comment:
what about `spark.warehouse.dir` ??

and what if `hive.metastore.warehouse.dir` is not configured ?

Reply:
Internally spark sets the value of spark.warehouse.dir to hive.metastore.warehouse.dir, so hive.metastore.warehouse.dir is enough here

Reply:
ok

Reply:
> Internally spark sets the value of spark.warehouse.dir to hive.metastore.warehouse.dir, so hive.metastore.warehouse.dir is enough here

we can mark this as comment in code